snappy-tools/src/main/scala/org/apache/spark/sql/columntable/ColumnFormatRelation.scala

```diff
@@ -264,8 +267,8 @@ class ColumnFormatRelation(
    */
   override def destroy(ifExists: Boolean): Unit = {
     // use a non-pool connection for operations
-    val conn = JdbcUtils.createConnection(externalStore.connProperties.url,
-      externalStore.connProperties.connProps)
+    val conn = JdbcUtils.createConnectionFactory(externalStore.connProperties.url,
```

Comment:
@rishitesh See the comment before. Better to create the connection factory once and keep as a private member in the class.